rails/app/controllers/api/v1/students_controller.rb

```diff
@@ -115,6 +115,15 @@ def register
     if user_attributes[:last_name].nil?
       return error("Missing user last_name parameter")
     end
+    if user_attributes[:password].nil?
```

Comment:
Shouldn't these be `.blank?` instead of `.nil?`

See https://stackoverflow.com/questions/885414/how-to-understand-nil-vs-empty-vs-blank-in-ruby.